recipes/prgenv-gnu/25.7/amdgpu/environments.yaml

```diff
@@ -0,0 +1,70 @@
+gcc-env:
+  compiler: [gcc]
+  network:
+      mpi: cray-mpich@8.1.32+rocm
+      specs: [ 'libfabric@2.3 +rocm' ]
+  unify: when_possible
+  specs:
+  - boost +chrono +filesystem +iostreams +mpi +python +regex +serialization +shared +system +timer
+  - cmake
+  - fftw
+  - fmt
+  - gsl
+  - hdf5+cxx+hl+fortran
+  - netlib-scalapack
+  - lua
+  - libtree
+  - lz4
+  - meson
+  - ninja
+  - openblas threads=openmp
+  #- osu-micro-benchmarks@5.9
+  - python@3
+  - zlib-ng
+  # add GPU-specific packages here, for easier comparison with mc version
+  - hip@6.3.3 ^mesa@23.3.6
+  - hdf5+mpi
+  - hipcub
+  - llvm-amdgpu
+  - llvm@18 +libomptarget +libomptarget_debug
```

Comment:
Just for my knowledge: is 18 chosen because it's the same version llvm-amdgpu@6.3 is based on or "arbitrary"?